src/realtime_qc.bib

```diff
@@ -0,0 +1,6 @@
+
+@article{johnson2021bgc,
+  title={BGC-Argo quality control manual for nitrate concentration},
+  author={Johnson, Kenneth and Maurer, Tanya and Plant, Joshua and Bittig, Henry and Schallenberg, Christina and Schmechtig, Catherine},
+  year={2021}
```

Comment:
I recommend we add the doi for this resource: 10.13155/84370

Reply:
Yes, good idea!